linode/ansible_linode
# Inject documentation before release using `ansible-specdoc`

## 📝 Description

**What does this PR do and why is this change necessary?**

Sets up injection step in release workflow to inject documentation into the module python files, required for ansible documentation and ansible_linode v1.0

**NOTE**: We're not entirely sure if this will render correctly in the ansible website, if it doesn't we'll have to disable this step and make an update to ansible-specdoc. Because of this I dont think the initial release including this PR should be v1.0.0 so we dont have to roll it back or jump straight to v1.0.1 like we have in the past.

Requires https://github.com/linode/ansible-specdoc/pull/38

## ✔️ How to Test

1. Use the make shortcut to inject each file
```bash
make deps
make inject
```
2. View a file in the `/plugins/modules/` folder and see the injected documentation.
